tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -178,21 +187,32 @@ def create_private_release_files(self):
             container_client = self.blob_service_client.get_container_client(
                 "releases")
 
-        self.upload_files(container_client,
-                          self.data_files)
-        self.upload_files(container_client,
-                          self.metadata_files)
+        try:
+            self.upload_files(container_client,
+                              self.data_files)
+        except ResourceExistsError:
+            pass
 
-    def upload_files(self, container_client, files):
-        data = b'abcd'*128
+        try:
+            self.upload_files(container_client,
+                              self.metadata_files)
+        except ResourceExistsError:
+            pass
+
+    @staticmethod
+    def upload_files(container_client, files):
+        data = b'abcd' * 128
         for file in files:
             path = file.path()
-            blob_client = container_client.get_blob_client(path)
-            blob_client.upload_blob(
-                data, blob_type="BlockBlob", metadata=file.metadata())
+            try:
+                blob_client = container_client.get_blob_client(path)
```

Comment:
Is it the `get_blob_client` or the `upload_blob` that raises the `ResourceExistsError`? Just wondering if `get_blob_client` needs to be within the `try`, if it's only `upload_blob`  that raises it.

Reply:
I'm a bit confused because this comment is resolved but I don't see any changes in the code or a reply here to explain. I think your safe to raise `blob_client = container_client.get_blob_client(path)` up one line in the code above the `try`.

Reply:
sorry about this, I read this PR comment the wrong way